.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "standard",
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
```

```suggestion
    "rules": {
      "no-unused-vars": [
        "error",
        {
          "argsIgnorePattern": "^_",
          "varsIgnorePattern": "^_",
          "caughtErrorsIgnorePattern": "^_"
        }
      ],
      "no-undef": ["error"],
      "prefer-const": ["error"],
      "no-var": ["error"],
      "no-ternary": "error",
      "no-return-await": "error",
      "no-plusplus": "error",
      "eqeqeq": ["error", "always"],
      "quotes": ["error", "double"],
      "semi": ["error", "always"]
    }
  }
```

these are some pretty basic (non-strict) rules, if you want to find more please visit [this](https://eslint.org/docs/latest/rules/)

---

Sure I will look into this, Thanks for mentioning it!

---

looks good, you may resolve this thread